packages/dx-react-grid/src/plugins/local-paging.test.jsx

```diff
@@ -81,8 +81,8 @@ describe('LocalPaging', () => {
       </PluginHost>,
     );
 
-    expect(defaultDeps.action.setCurrentPage.mock.calls)
-      .toEqual([[2]]);
+    expect(defaultDeps.action.setCurrentPage.mock.calls[0][0])
+      .toEqual(2);
```

Comment:
Why not use the `.toHaveBeenCalledWith()` matcher?

Reply:
Action is called with additional arguments (getters and actions)